### ISSUE TITLE
Install Redis Stack on Linux Support for linuxkit

### ISSUE THREAD

**Issue by rapito** [redis-stack/redis-stack-docs#48]
Hi, we currently use a distribution of linux called linuxkit (5.10.76-linuxkit) which is the same base image that most CircleCI images are based off.
I can install pretty much every service in here but when I execute these commands, it will error on package not found during the `apt-get install` command:

```
curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg
echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list
sudo apt-get update
sudo apt-get install redis-stack-server
```

I also attempted to use the unpacked binaries and followed this example from https://redis.io/docs/stack/get-started/install/binaries/ but it errored out on missing libraries as well as missing GLIBC_2.29 which is not yet available on this distro.
```
Starting redis-stack-server, database path /var/lib/redis-stack
./bin/redis-server: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./bin/redis-server)
./bin/redis-server: /usr/lib/x86_64-linux-gnu/libssl.so.1.1: version `OPENSSL_1_1_1' not found (required by ./bin/redis-server)
./bin/redis-server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by ./bin/redis-server)
```

Given that running any version of redis is possible under this distribution of linux, would it be possible to provide a deb package that can be installed as well?

**Comment by rapito** [comment]
Hi @chayim, our images are based off `circleci/ruby:2.6.5-stretch-node` https://hub.docker.com/r/circleci/ruby/tags?page=1&name=2.6.5-stretch-node. You can reference it from docker hub.

**Comment by chayim** [comment]
So I've got good news, and bad news sports fans... We don't currently build on Debian Stretch.  I took the Ubuntu xenial binaries are tried to make those work *but* unfortunately there's a libssl mismatch. The bionic binaries have the glibc mismatch issue.

On the bright side, I'm about to release versions that support Debian bullseye - and [circleci/ruby:2.6.9-bullseye-node](https://hub.docker.com/layers/ruby/circleci/ruby/2.6.9-bullseye-node/images/sha256-52b4fb7f272a5073ce069a2f49eddd49da7e548daaafb698764baaf773ade08b?context=explore) exists.

Any chance that might your need? I'm... hoping!

**Comment by chayim** [comment]
Ping @rapito ... I'd love to get this in the end of June release...

**Comment by rapito** [comment]
Hi @chayim sorry I missed the ping
We managed to use a redis + redis stack image that we could spawn up on circle ci with already exposed ports:
`image: redis/redis-stack:latest`

Seems like the new dependency on GLIBC version is making noise on older linux distros as well...
Since we managed to fill our need, I'm closing this now.

I'm still noting the debian bullseye support, that's very nice!
Thanks for the help!
